### DISCUSSION TITLE
Index id not defined error

### DISCUSSION THREAD

**Discussion by vhdm** [livewire/livewire#3763] [7. Bugs (archived)] [answered]
![image](https://user-images.githubusercontent.com/14149777/133502004-7ddf5bd7-c765-4e57-b536-4a80614c8f77.png)

**Comment by joshhanley** [comment, marked as answer]
@vhdm please see this comment for a solution https://github.com/livewire/livewire/discussions/3724#discussioncomment-1332569 🙂
